src/9-v04x-migration/index.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2016-02-12
+id: 9-v04x-migration
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "9-v04x-migration", link: "./" }
+tags: gateway, bootstrap, infrastructure
+title: Migrating ipfs.io from go-ipfs 0.3.x to 0.4.0
+author: Lars Gierth
+collection: posts
+---
+
+Good news everyone! We are about to release go-ipfs 0.4.0,
+which contains lots of great changes and enhancements.
+
+There are breaking changes
+which prevent 0.4.x nodes from communicating with 0.3.x nodes.
+This means that there will be, and already are, two separate networks.
+We'll call these networks v03x and v04x.
+IPFS nodes built from the master branch are already part of v04x.
+The Docker image `ipfs/go-ipfs` is built from master and thus also part of v04x.
+
+ipfs.io provides two essential services to the community,
+which are affected by this: the gateway and the default bootstrappers.
+We'll continue to support them for the v03x network until the **end of April 2016**.
+
+Please note that we won't support 0.3.x with patches or new features.
+All development effort is directed towards 0.4.0, and you should update as soon as possible.
+
+- [How do I update go-ipfs to 0.4.0?](#how-do-i-update-go-ipfs-to-0-4-0-)
+- [The public gateways at ipfs.io](#the-public-gateways-at-ipfs-io)
+- [The default bootstrappers](#the-default-bootstrappers)
+- [How does this affect me?](#how-does-this-affect-me-)
+
+
+## How do I update go-ipfs to 0.4.0?
+
+You can use the [ipfs-update tool][ipfs-update] to update go-ipfs.
```

Comment:
You can: 
- use the ipfs-update tool to update go-ipfs **(recommended)**
- download [the new go-ipfs binaries][go-ipfs-bins] and install them.
- download [the new source](https://github.com/ipfs/go-ipfs) and [build it yourself](https://github.com/ipfs/go-ipfs#build-from-source).

Please note that installing with `go get github.com/ipfs/go-ipfs` does not work at this time. We are experimenting with our [ipfs-based package manager](https://github.com/whyrusleeping/gx).